image/cli/mascli/templates/pipelinerun.yaml

```diff
@@ -34,6 +34,16 @@ spec:
       value: '$DB2_CPU_REQUESTS'
     - name: db2_cpu_limits
       value: '$DB2_CPU_LIMITS'
+    - name: db2_meta_storage_size
```

Comment:
Where do these new values get set in non-SNO mode?  `config_pipeline_storage_db2_size` only seems to do anything whern SNO mode == true.